addons/mail/static/src/components/attachment_list/attachment_list.scss

```diff
@@ -6,8 +6,11 @@
     display: flex;
     flex-flow: wrap;
     justify-content: flex-start;
+    margin: map-get($spacers, 1);
 }
 
 .o_AttachmentList_attachment {
-    margin: map-get($spacers, 1);
+    margin: map-get($spacers, 1) 0;
+    margin-inline-end: map-get($spacers, 1);
```

Comment:
I would avoid mixing these 2 rules that override each other, priority is not clear to me especially with "inline-end" and "margin" itself including "left and right".

Just explicit each case if necessary, but it would be best if just `margin: map-get($spacers, 1);` was good for all?